src/Microsoft.DotNet.Wpf/ApiCompat/Baselines/PresentationBuildTasks-Net48.baseline.txt

```diff
@@ -0,0 +1,3 @@
+Compat issues with assembly PresentationBuildTasks:
```

Comment:
so we shouldn't expect any further changes to these files between now and RTM, correct?

Reply:
These baselines are always updated manually (usually by building with a specific Arcade flag set).  We don't expect there to be any ApiCompat breaks that will require us to do that though.